URBNCreativeTestRig/FontsTableViewController.swift

```diff
@@ -0,0 +1,104 @@
+//
+//  FontsTableViewController.swift
+//  URBNCreativeTestRig
+//
+//  Created by Jason Grandelli on 11/12/15.
+//  Copyright © 2015 Jason Grandelli. All rights reserved.
+//
+
+import UIKit
+import URBNDataSource
+
+class FontsTableViewController: UITableViewController, UISearchResultsUpdating {
+
+    private let completion: (fontName: String) -> (Void)
+    private let dataSource = URBNArrayDataSourceAdapter(items: nil)
+    private var tableData: Array<String> = []
+    private lazy var typeFaces: Array<String> = {
+        var fonts: Array<String> = []
```

Comment:
Most of the apple documentation I've seen is using the newer syntax i.e. `[ObjectType]`. I'm not crazy about it but I've resigned to using it myself to using it to normalize our code a bit.

By the way I really don't like how Swift lets you do things like that multiple ways because it creates these discussions.